packages/components/src/custom-gradient-picker/index.tsx

```diff
@@ -148,19 +147,19 @@ export function CustomGradientPicker( {
 	onChange,
 	__experimentalIsRenderedInSidebar = false,
 }: CustomGradientPickerProps ) {
-	const { gradientAST, gradientAstValue } =
-		getGradientAstWithDefault( value );
+	const { gradientAST, hasGradient } = getGradientAstWithDefault( value );
+
 	// On radial gradients the bar should display a linear gradient.
 	// On radial gradients the bar represents a slice of the gradient from the center until the outside.
 	// On liner gradients the bar represents the color stops from left to right independently of the angle.
 	const background = getLinearGradientRepresentation( gradientAST );
-	const hasGradient = gradientAstValue !== DEFAULT_GRADIENT;
+
 	// Control points color option may be hex from presets, custom colors will be rgb.
 	// The position should always be a percentage.
 	const controlPoints = gradientAST.colorStops.map( ( colorStop ) => {
 		return {
 			color: getStopCssColor( colorStop ),
-			// Although it's already been checked by `hasUnsupportedLength` in `getGradientAstWithDefault`,
+			// Although it's already been checked by `hasUnsupportedLength` in `getGradientAst`,
```

Comment:
I notice you've changed this comment, but the function returning `gradientAST` is still named `getGradientAstWithDefault`. Did you mean to rename the function now that it's handling default gradients differently?

Reply:
I did rename the function at some point, but then I changed my mind and restored the original name. Although I forgot to update the comment — thank you for catching it!